src/MongoDB/Server.c

```diff
@@ -384,6 +385,23 @@ static PHP_METHOD(Server, getPort)
 	phongo_throw_exception(PHONGO_ERROR_RUNTIME, "Failed to get server description");
 } /* }}} */
 
+static PHP_METHOD(Server, getServerDescription)
+{
+	zend_error_handling  error_handling;
+	php_phongo_server_t* intern;
+
+	intern = Z_SERVER_OBJ_P(getThis());
+
+	zend_replace_error_handling(EH_THROW, phongo_exception_from_phongo_domain(PHONGO_ERROR_INVALID_ARGUMENT), &error_handling);
+	if (zend_parse_parameters_none() == FAILURE) {
+		zend_restore_error_handling(&error_handling);
+		return;
+	}
+	zend_restore_error_handling(&error_handling);
```

Comment:
This was only first used in ServerApi and not changed in existing files, but the `PHONGO_PARSE_PARAMETERS_NONE` macro simplifies this:
```suggestion
	php_phongo_server_t* intern = Z_SERVER_OBJ_P(getThis());

	PHONGO_PARSE_PARAMETERS_NONE();
```